src/views/profile/Deploy.vue

```diff
@@ -0,0 +1,234 @@
+<script setup lang="ts">
```

Comment:
Please don't extend PR's, otherwise we will review this forever ;) This one was about upload, Deploy should be in separate one.

Reply:
My bad, I mistakenly merged with my deploy local branch